sr_demos/scripts/sr_demos/demo.py

```diff
@@ -372,30 +372,34 @@ def sequence_lf(hand_commander, joint_states_config, inter_time_max, tactile_rea
             # is not in the middle of a movement, generate a random position
             # and interpolation time
             else:
-                complete_random_sequence(wake_time, hand_commander, joint_states_config, inter_time_max)
+                if time.time() < wake_time + 15:
```

Comment:
what is this 15? Please put that in a const variable at least so there are no "magic numbers" floating around